libafl_frida/Cargo.toml

```diff
@@ -82,6 +82,7 @@ log = "0.4.20"
 mmap-rs = "0.6.0"
 
 yaxpeax-arch = "0.2.7"
+xxhash-rust = { version = "0.8.5", features = ["xxh3"]} # xxh3 hashing for rust
```

Comment:
We try to get around xxxhash-rust as mandatory dependency, can we optionally use ahash here?